jobs/pull-kubernetes-e2e-kops-aws.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+readonly testinfra="$(dirname "${0}")/.."
+
+# TODO(fejta): remove this
+case "${ghprbTargetBranch:-}" in
+  "release-1.0"|"release-1.1"|"release-1.2"|"release-1.3")
+    echo "PR AWS kops job disabled for legacy branches."
+    exit
+    ;;
+esac
+
+export KUBE_GCS_RELEASE_BUCKET="${KUBE_GCS_RELEASE_BUCKET:-kubernetes-release-pull}"
+export KUBE_GCS_RELEASE_SUFFIX="/${JOB_NAME}"
+export KUBE_GCS_UPDATE_LATEST=n
+export JENKINS_USE_LOCAL_BINARIES=y
+export KUBE_FASTBUILD=true
+
+if [[ -z "${SKIP_BUILD:-}" ]]; then
+  ./hack/jenkins/build.sh
+fi
+
+readonly version=$(source build-tools/util.sh && echo $(kube::release::semantic_version))
+export KUBERNETES_PROVIDER="kops-aws"
+
+export KOPS_STATE_STORE="${KOPS_STATE_STORE:-s3://k8s-kops-jenkins/}"
+export KOPS_CLUSTER_DOMAIN="${KOPS_CLUSTER_DOMAIN:-test-aws.k8s.io}"
+export E2E_NAME="aws-kops-${NODE_NAME}-${EXECUTOR_NUMBER:-0}"
+export E2E_OPT="${E2E_OPT:-}\
+  --kops-cluster ${E2E_NAME}.${KOPS_CLUSTER_DOMAIN}\
+  --kops-kubernetes-version https://storage.googleapis.com/${KUBE_GCS_RELEASE_BUCKET}/ci${KUBE_GCS_RELEASE_SUFFIX}/${version}\
+  --kops-nodes 4\
```

Comment:
Where did you see the other? I bumped https://github.com/kubernetes/test-infra/pull/979/files#diff-cc56a431187d0d372655f10b42900f84, but that's the normal job.